### DISCUSSION TITLE
No Metadata saving anymore?

### DISCUSSION THREAD

**Discussion by carlmoss22** [MoonRide303/Fooocus-MRE#178] [Q&A]
Hi guys, 
is there no metadata saving anymore? No button to see and no data saved.

**Comment by barepixels** [comment]
confirm you are using MRE
check Misc tab

**Reply by carlmoss22** [reply]
oooooooh, i mixed it up. it's the original one!

thanx for answer!

**Comment by Ypdes** [comment]
I checked the "Output Format" as "png" and "Save metadata in image", but the images still don't contain metadata. Why?
